pkgs/development/python-modules/ospd/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, defusedxml
+, deprecated
+, fetchFromGitHub
+, lxml
+, paramiko
+, poetry
+, psutil
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "ospd";
+  version = "21.4.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
```

Comment:
```suggestion
  disabled = pythonOlder "3.7" || stdenv.isDarwin;
```

Reply:
Is it preferable to do this via disabled or by setting platforms on the metadata field?

Reply:
I am not sure but now I would rather use meta.platforms because disabled is points to the python interpreter if something is now allowed.